src/js/components/atoms/c-image/data/examples/nylon.ts

```diff
@@ -0,0 +1,5 @@
+export const nylon = {
+    alt: 'Nylon 3D printed gears',
+    mimeType: 'image/jpeg',
+    url: 'https://ultimaker.com/img/materials/carousel/Nylon/Nylon-3D-printed-gears.jpg',
```

Comment:
These images are not hosted on contentful, so it can not be resized/cropped with the url properties generated by c-image. (the /generator/svg, can because the properties are mocked). This also counts for though pla mockdata.